src/cli/commands/files/get.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+const Command = require('ronin').Command
+const debug = require('debug')
+const utils = require('../../utils')
+const log = debug('cli:files')
+log.error = debug('cli:files:error')
+var fs = require('fs')
+const pathj = require('path')
+
+module.exports = Command.extend({
+  desc: 'Download IPFS objects',
+
+  run: (path, outPath) => {
+    let dir
+    let filepath
+    let ws
+
+    if (!path) {
+      throw new Error("Argument 'path' is required")
+    }
+    if (!outPath) {
+      outPath = {}
+      dir = process.cwd()
+    } else {
+      if (outPath.slice(-1) !== '/') {
```

Comment:
I do this once, and it is an attempt to be like go-ipfs. It is checking to see if you supply a path with either / on the end or without. If you don' t supply a / then one needs to appended to the supplied path. It's just so things don't break if for example you do this 

```
jsipfs files get QmTHwJEAryXDVmNWZYRvJBBdMjza7ZEgbcpUVwcRY5TsUi /home/n4th4n/Documents/test
```
instead of one with a / at the end like 

```
jsipfs files get QmTHwJEAryXDVmNWZYRvJBBdMjza7ZEgbcpUVwcRY5TsUi /home/n4th4n/Documents/test/
```